### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.5 - es. 3.15 (21 ottobre)

### DISCUSSION THREAD

**Discussion by alem1105** [sapienzastudentsnetwork/calcolo-delle-probabilita#298] [Esercizi M-Z]
<img width="954" alt="immagine" src="https://github.com/user-attachments/assets/86918782-4700-41d3-97fc-a4df931ffbe1">

**Comment by alem1105** [comment]
![image](https://github.com/user-attachments/assets/404e4ce8-29c4-4a7b-a5e3-476a0fec557c)

**Comment by AglaiaNorza** [comment]
![image](https://github.com/user-attachments/assets/86a40f3a-597f-4293-81de-b19c08474e62)

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/be47ed31-86c8-4861-87a6-9fd1cfc3e68c)
